test/e2e/example_k8petstore.go

```diff
@@ -150,7 +150,7 @@ T:
 	Ω(totalTransactions).Should(BeNumerically(">", minExpected))
 }
 
-var _ = Describe("[Example] Pet Store [Skipped]", func() {
+var _ = Describe("Pet Store [Feature:Example]", func() {
```

Comment:
Feature tests are skipped by default:

https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/e2e.sh#L272

and

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/e2e_test.go#L151

But it looks like I screwed up the `[Feature]` thing in `e2e_test.go`.  Will send PR shortly.